02week/ticTacToe.js

```diff
@@ -0,0 +1,167 @@
+'use strict';
+
+const assert = require('assert');
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+let board = [
+  [' ', ' ', ' '],
+  [' ', ' ', ' '],
+  [' ', ' ', ' ']
+];
+
+let playerTurn = 'X';
+
+function printBoard() {
+  console.log('   0  1  2');
+  console.log('0 ' + board[0].join(' | '));
+  console.log('  ---------');
+  console.log('1 ' + board[1].join(' | '));
+  console.log('  ---------');
+  console.log('2 ' + board[2].join(' | '));
+}
+
+//my code below
+
+function horizontalWin() {
+  for(board[i] = 0; i < 3; i++) {
+    if (board[i] === ['X', 'X', 'X'] || ['O','O','O']) {
+        return true;
+    } 
+  }
+//written out as if 
+//  if ([0] === ['X', 'X', 'X'] || ['O','O','O']) {
+//    return "Winner!"
+//  }
+ // else if ([1] === ['X', 'X', 'X'] || ['O','O','O']) {
+//    return "Winner!"
+//  }
+  //else if ([2] === ['X', 'X', 'X'] || ['O','O','O']) {
+    //return "Winner!"
+ // }
+}
+
+function verticalWin() {
+  for(board[i][0] = 0; i < 3 ; i++) {
+    if (board[i][0] === 'X'){
+      return true;
+    } else if (board[i][0] === 'O'){
+      return true;
+    }
+  }
+
+  for(board[i][1] = 0; i < 3 ; i++) {
+    if (board[i][1] === 'X'){
+      return true;
+    } else if (board[i][1] === 'O'){
+      return true;
+    }
+  }
+
+  for(board[i][2] = 0; i < 3 ; i++) {
+    if (board[i][2] === 'X'){
```

Comment:
checking for one cell at a time and returning true would cause you to say someone won if there is an 'X' in any cell, but you want to check if the entire row has x's